values.yaml

```diff
@@ -460,6 +472,10 @@ mail:
       readiness: {}
       startup: {}
       liveness: {}
+  
+  # Mail relay specific labels
+  labels:
```

Comment:
LGTM but I'm going to move this under `relay`, with the rest of the deployment-specific fields.